### ISSUE TITLE
[Web] Keyframe does not work on the web

### ISSUE THREAD

**Issue by yatessss** [software-mansion/react-native-reanimated#3549]
### Description

example in doc https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/keyframeAnimations#example, does not work on web (no animation).

but work fine in native platform.

```js
export function KeyframeAnimation(): React.ReactElement {
  const [show, setShow] = useState(false);

  const enteringAnimation = new Keyframe({
    0: {
      originX: 50,
      transform: [{ rotate: '45deg' }],
    },
    30: {
      originX: 10,
      transform: [{ rotate: '-90deg' }],
    },
    100: {
      originX: 0,
      transform: [{ rotate: '0deg' }],
      easing: Easing.quad,
    },
  }).duration(2000);

  const exitingAnimation = new Keyframe({
    0: {
      opacity: 1,
      transform: [{ skewX: '0deg' }],
    },
    30: {
      opacity: 0.5,
      transform: [{ skewX: '40deg' }],
      easing: Easing.exp,
    },
    100: {
      opacity: 0,
      transform: [{ skewX: '-10deg' }],
    },
  }).duration(2000);
  
  return (
    <View style={{ flexDirection: 'column-reverse' }}>
      <Button
        title="animate"
        onPress={() => {
          setShow((last) => !last);
        }}
      />
      <View
        style={{ height: 400, alignItems: 'center', justifyContent: 'center' }}>
        {show && (
            <Animated.View
              entering={enteringAnimation}
              exiting={exitingAnimation}
              style={{
                height: 100,
                width: 200,
                backgroundColor: 'blue',
                alignItems: 'center',
                justifyContent: 'center',
              }}
            />
        )}
      </View>
    </View>
  );
}
```

### Steps to reproduce

copy code in project and run

### Snack or a link to a repository

no

### Reanimated version

2.9.0

### React Native version

0.69.0

### Platforms

Web

### Acknowledgements

Yes

**Comment by tomekzaw** [comment]
Hey @yatessss, thanks for submitting this issue.

Unfortunately, Layout Animations are not supported on Web yet. 😢

**Comment by fo-fo** [comment]
Any plans to add layout animations for web?

Also it might be a good idea to add a blurb to https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/web-support to explain that it's not supported at this time.

**Comment by tomekzaw** [comment]
> Any plans to add layout animations for web?

Yes, but first we need to finish refactoring Layout Animations for Android & iOS which is currently in progress.

> Also it might be a good idea to add a blurb to https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/web-support to explain that it's not supported at this time.

That's right, thanks for the suggestion!

**Comment by David06A** [comment]
Hey! Any news or a place I can check for any regarding keyframes on web

**Comment by TNAJanssen** [comment]
Yeah this feature would be really nice!

**Comment by Akronae** [comment]
Also looking forward to it. Might be a good idea to specify in [the docs](https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/keyframeAnimations/) that it is currently not available for web.

**Comment by jaivinwylde** [comment]
What's the timeline for this? How much of the refactor for mobile layout animations have been completed @tomekzaw?

**Comment by jaivinwylde** [comment]
?

**Comment by tomekzaw** [comment]
Hey, Layout Animations for mobile have been finished and released in [3.0.0-rc.6](https://github.com/software-mansion/react-native-reanimated/releases/tag/3.0.0-rc.6).

**Comment by yatessss** [comment]
> Hey, Layout Animations for mobile have been finished and released in [3.0.0-rc.6](https://github.com/software-mansion/react-native-reanimated/releases/tag/3.0.0-rc.6).

Thank you for your help!

**Comment by Veeeeksi** [comment]
Dunno, why this is closed because Keyframe animations still won't work on web but I also look forward to see them in future so I don't have to implement separate animations for web + mobile!
